src/get-started/wordcount-example.md

```diff
@@ -698,22 +829,33 @@ public static void main(String[] args) throws IOException {
 ```
 
 ```py
-This feature is not yet available in the Beam SDK for Python.
+# This feature is not yet available in the Beam SDK for Python.
 ```
 
 ### Adding Timestamps to Data
 
-Each element in a `PCollection` has an associated **timestamp**. The timestamp for each element is initially assigned by the source that creates the `PCollection` and can be adjusted by a `DoFn`. In this example the input is bounded. For the purpose of the example, the `DoFn` method named `AddTimestampsFn` (invoked by `ParDo`) will set a timestamp for each element in the `PCollection`.
-  
+Each element in a `PCollection` has an associated **timestamp**. The timestamp
+for each element is initially assigned by the source that creates the
+`PCollection` and can be adjusted by a `DoFn`. In this example the input is
```

Comment:
(with some caveats - specifically, it timestamps can by default only be moved forwards in time, and some sources let the user attach a timestamp at the site of the read)

Reply:
changed wording